nservicebus/operations/tuning.md

```diff
@@ -8,34 +8,32 @@ redirects:
  - nservicebus/how-to-reduce-throughput-of-an-endpoint
  - nservicebus/operations/reducing-throughput
  - nservicebus/operations/throughput
-reviewed: 2020-06-23
+reviewed: 2021-11-01
 ---
 
 NServiceBus uses defaults that ensure good performance in common cases. While this is usually the preferred mode of operation there are situations where tuning might be desired.
 
-## Tuning concurrency
+Examples where concurrency tuning might be relevant are:
+
+ * Non thread safe code that needs to run sequentially
+ * Databases that might deadlock when getting too many concurrent requests
+ 
+## Configuring concurrency limit
 
 partial: defaults
 
-Limit maximum concurrency so that no more messages than the specified value are ever processed at the same time. If a maximum concurrency is not specified, the transport will choose an optimal value that is a balance between throughput and effective resource usage.
+Limit maximum concurrency so that no more messages than the specified value are ever processed at the same time. If a maximum concurrency is not specified, the transport will choose an optimal value that is a balance between throughput and effective resource usage. The concurrency limit cannot be changed at run-time and can only be applied at endpoint instance creation and requires the instanceto be restarted for concurrency changes to take effect.
```

Comment:
```suggestion
Limit maximum concurrency so that no more messages than the specified value are ever processed at the same time. If a maximum concurrency is not specified, the transport will choose an optimal value that is a balance between throughput and effective resource usage. The concurrency limit cannot be changed at run-time and can only be applied at endpoint instance creation and requires the instance to be restarted for concurrency changes to take effect.
```